ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/avd_dir_mode.yml

```diff
@@ -0,0 +1,2 @@
+avd_file_mode: 0o666
+avd_dir_mode: 0o666
```

Comment:
make sure you have x (LSB) set on avd_dir_mode. Otherwise you cannot do operations on the directory. So test with `avd_dir_mode: 0o777`